gui/src/components/s3/iam-user-management.vue

```diff
@@ -628,7 +637,8 @@ export default class CortxIAMUserManagement extends Vue {
     this.user = {} as IAMUser;
   }
 
-  public async mounted() {
+  public async mounted() {    
+    this.checkPermissions();
```

Comment:
Please use await as checkPermissions is an async function.

Reply:
await is added